terraform/main.tf

```
@@ -320,6 +341,148 @@ resource "aws_iam_role_policy" "api_task-publish_to_arpa_audit_report_queue" {
   policy      = data.aws_iam_policy_document.publish_to_arpa_audit_report_queue.json
 }
 
+data "aws_iam_policy_document" "publish_to_digest_email_get_grants" {
+  statement {
+    sid       = "AllowPublishToQueue"
+    actions   = ["sqs:SendMessage"]
+    resources = [module.digest_email_get_grants.sqs_queue_arn]
+  }
+}
+
+resource "aws_iam_role_policy" "digest_email_kickoff_cron-publish_to_digest_email_get_grants_queue" {
+  name_prefix = "send-digest-email-get-grants-requests"
+  role        = module.api.digest_email_kickoff_cron_ecs_task_role_name
```

Per item 2b in our [Slack discussion](https://usdigitalresponse.slack.com/archives/C0324KDQSCR/p1699579161642309?thread_ts=1699568729.822169&cid=C0324KDQSCR):

```suggestion
  role        = module.api.ecs_task_role_name
```